### ISSUE TITLE
MultimodalAgent error on participant rejoin

### ISSUE THREAD

**Issue by kristina-fefelova** [livekit/agents-js#214]
I'm trying to use `MultimodalAgent` and was faced with the problem:
if participants rejoin in the same room, I see theese error
```
node_modules/.pnpm/@livekit+agents@0.5.1_@livekit+rtc-node@0.12.1/node_modules/@livekit/agents/dist/multimodal/multimodal_agent.js:306
          reject(new Error("Task cancelled"));
```
This happens due to the repeated event `RoomEvent.Track Subscribed`

After this error the agent stops responding

**Comment by Ali-Raza995** [comment]
@kristina-fefelova Is it resolved ? or you are facing the issue?

plus are you facing this kind of this kind of issue ?
SyntaxError: The requested module '@bufbuild/protobuf' does not provide an export named 'Duration'

**Comment by Ali-Raza995** [comment]
@kristina-fefelova previously it was working perfectly. but now i am getting this error may be because versions updated.
SyntaxError: The requested module '@bufbuild/protobuf' does not provide an export named 'Duration'.

currently in package.json I am having this 


  "dependencies": {
    "@bufbuild/protobuf": "^2.2.3",
    "@livekit/agents": "^0.3.5",
    "@livekit/agents-plugin-openai": "^0.3.5",
    "@livekit/rtc-node": "^0.11.0",
    "@pinecone-database/pinecone": "^4.0.0",
    "axios": "^1.7.8",
    "eslint": "^8",
    "eslint-config-prettier": "^9.1.0",
    "eslint-plugin-prettier": "^5.2.1",
    "eslint-plugin-unused-imports": "4.1.4",
    "firebase-admin": "^13.0.1",
    "googleapis": "^144.0.0",
    "openai": "^4.73.1",
    "pnpm": "^9.13.2",
    "zod": "^3.23.8"
  },
  
  although we dont need of this   "@bufbuild/protobuf": "^2.2.3".
  
  did you faced the same problem?

**Comment by Ali-Raza995** [comment]
@kristina-fefelova Thanks. My issue fixed with this  
 "pnpm": {
    "overrides": {
      "livekit-server-sdk": "2.8.1"
    }
  },

**Comment by nbsp** [comment]
fixed in 0.6.0
